modules/widgets/google-translate/google-translate.js

```diff
@@ -0,0 +1,20 @@
+/*global google:true*/
+/*global _wp_google_translate_widget:true*/
+/*exported googleTranslateElementInit*/
+function googleTranslateElementInit() {
+	var lang = 'en';
+	var langParam;
+	var langRegex = /[?&#]lang=([a-z]+)/;
+	if ( typeof _wp_google_translate_widget === 'object' && typeof _wp_google_translate_widget.lang === 'string' ) {
+		lang = _wp_google_translate_widget.lang;
+	}
+	langParam = window.location.href.match( langRegex );
```

Comment:
Declarations can be written:
```js
var langRegex = /[?&#]lang=([a-z]+)/,
    langParam = window.location.href.match( langRegex ),
    lang = 'object' === typeof _wp_google_translate_widget && 'string' === typeof _wp_google_translate_widget.lang ? _wp_google_translate_widget.lang : 'en';
```
to avoid redundancy.